src/tests/api/session.test.ts

```diff
@@ -0,0 +1,67 @@
+import { assert200OK, assert400BadRequest, assert401Unauthorized } from "../../utils/apiCommonAssertions";
+import { sessionPost, stubStartPost } from "../../utils/apiTestSteps";
+
+describe("Happy Path Get SessionID", () => {
+	let stubResponse: any;
+
+	beforeEach( async () =>{
+		const requestBody = {
+			target:"https://unb2uhs7rl.execute-api.eu-west-2.amazonaws.com/dev/",
+		};
+		stubResponse = await stubStartPost(requestBody);
+
+	});
+    
+	it("Given valid endpoint has been received", async () => {
```

Comment:
Are there permutations of JAR objects we accept?

our default one has shared claims, do we still get successfully create a session with no shared claims?